toolkit/scripts/check_static_glibc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyrpm.spec import Spec
+from os.path import dirname, realpath
+
+import argparse
+import re
+import sys
+
+def get_glibc_version():
+    glibc_spec = Spec.from_file("SPECS/glibc/glibc.spec")
+    epoch_prefix = "{}:".format(glibc_spec.epoch) if glibc_spec.epoch else ""
+    release_without_dist = glibc_spec.release.replace("%{?dist}", "")
+    release_suffix = "-{}".format(release_without_dist)
+    glibc_version = epoch_prefix + glibc_spec.version + release_suffix
+    return glibc_version
+
+def check_spec(path, glibc_version):
+    spec = Spec.from_file(path)
+    for br in spec.build_requires:
+        if br.name == "glibc-static":
+            issues = []
+            if not br.version:
+                issues.append(f"  * Does not specify a minimum version for its `glibc-static` BuildRequires")
+            if br.version and br.version != glibc_version:
+                issues.append(f"  * Does not specify the latest `glibc-static` version in its BuildRequires")
```

Comment:
Nit-pick: I'd avoid f-strings when we use plain strings, so that it's clear there's nothing special happening here.

```suggestion
                issues.append("  * Does not specify a minimum version for its `glibc-static` BuildRequires")
            if br.version and br.version != glibc_version:
                issues.append("  * Does not specify the latest `glibc-static` version in its BuildRequires")
```